### DISCUSSION TITLE
Different Image/PNG depending on color mode (dark/light)

### DISCUSSION THREAD

**Discussion by libellux** [vuejs/vitepress#3689] [Q&A] [answered]
Hello VitePress community,

I've tried different approaches (reading up) but haven't successfully been able to extend the VitePress **Default theme** to resolve my Question.

What I want to achieve is to show two different images within content/markdown depending on color mode (but also to update/switch on color or route change). If anyone has an example code snippet on how-to would be greatly appreciated.

Best regards,
Fredrik

**Comment by brc-dd** [comment, marked as answer]
https://github.com/vuejs/vitepress/discussions/3560
https://github.com/vuejs/vitepress/discussions/1130

**Reply by libellux** [reply]
Thanks for #3560 resolved it for me!
